build-tools/dependencies/dependencies.projitems

```diff
@@ -7,51 +7,61 @@
   <ItemGroup>
     <RequiredProgram Include="$(HostCcName)"        Condition=" '$(HostOS)' != 'Windows' " />
     <RequiredProgram Include="$(HostCxxName)"       Condition=" '$(HostOS)' != 'Windows' " />
-    <RequiredProgram Include="7za"                  Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="x86_64-w64-mingw32-g++" Condition=" '$(HostOS)' == 'Darwin' " >
+      <Homebrew>mingw-w64</Homebrew>
+    </RequiredProgram>
+    <RequiredProgram Include="7za"                  Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>p7zip</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' != 'Windows' ">
+    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' == 'Darwin' ">
       <Homebrew>autoconf</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="automake"             Condition=" '$(HostOS)' != 'Windows' ">
+    <RequiredProgram Include="automake"             Condition=" '$(HostOS)' == 'Darwin' ">
       <Homebrew>automake</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="cmake"                Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="cmake"                Condition=" '$(HostOS)' == 'Darwin'">
```

Comment:
Actually, given that `cmake` is now used in `src/monodroid`, isn't this required, full stop?  No `Condition` needed?

Reply:
Maybe. llvm uses cmake, but I don't think Mono SDKs use the cmake we install with Android SDK. As such, this check doesn't hurt.